### PR TITLE
Update the solution to the `-fPIC` problem.

### PR DESCRIPTION
**Pull request by lzlarryli**
The information was outdated. `hardening-wrapper` has been removed from ArchLinux for a while now.

Note: Documentation fixes for https://docs.haskellstack.org/en/stable/ should target the "stable" branch, not master.

Please include the following checklist in your PR:

* [ ] Any changes that could be relevant to users have been recorded in the ChangeLog.md
* [ ] The documentation has been updated, if necessary.

Please also shortly describe how you tested your change. Bonus points for added tests!

### CODE REVIEW

File: doc/faq.md [hunk 1]
```diff
@@ -434,13 +434,7 @@ such hardening flags by default which may be the cause of some instances of the
...
 flags.
 
-In Arch Linux, the support for this is provided by the `hardening-wrapper`
-package. Some possible workarounds:
-
-* Selectively disabling its PIE forcing by setting `HARDENING_PIE=0` in `/etc/hardening-wrapper.conf`.
-* Uninstalling the `hardening-wrapper` package and logging out then into your account again.
-
-If you manage to work around this in other distributions, please include instructions here.
+In Arch Linux, install the package `ncurses5-compat-libs` from AUR resolves this issue [see issue](https://github.com/commercialhaskell/stack/issues/2712).
```
Comment (lwm) [inline doc/faq.md]: I would recommend:

``` markdown
On Arch Linux, install the `ncurses5-compat-libs` package from AUR to 
resolve [this issue](https://github.com/commercialhaskell/stack/issues/2712).
```

Comment (lzlarryli) [reply doc/faq.md]: Grammar fixed:-)

Comment (lwm) [reply doc/faq.md]: Thanks!

**Comment by mgsloan** [comment]
Is it likely that people are using older versions of archilinux?  Might be good to still have the info for older versions.

**Comment by lwm** [comment]
Tried to bump the target branch to stable but Git said no :|

**Comment by mgsloan** [comment]
Oh yeah, good point, having the merge target be the `stable` branch would also be good.

**Comment by lzlarryli** [comment]
@lwm Hmmm..... I see the problem now. Let me just close this and resubmit using `stable` as base.

**Pull request closed without merging**